subprojects/packagefiles/libgpiod/bindings/cxx/meson.build

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+if add_languages('cpp', native: false)
+    abi_cxx_version = '2.2.1'
```

Comment:
add_languages() defaults to required: true, there's no need to put this in an if block since the build will abort if the C++ compiler is not found.

Also, if someone specifies -Dbindings=cxx then failure to configure the cxx bindings *should* be fatal. :p

Reply:
Yep, fair. I'll fix that.